contractcourt/commit_sweep_resolver.go

```diff
@@ -313,6 +330,19 @@ func newCommitSweepResolverFromReader(r io.Reader, resCfg ResolverConfig) (
 	return c, nil
 }
 
+// report returns a report on the resolution state of the contract.
+func (c *commitSweepResolver) report() *ContractReport {
+	c.reportLock.Lock()
+	defer c.reportLock.Unlock()
+
+	if c.currentReport == nil {
```

Comment:
having this be non-pointer we could just return it directly? Assuming it is always populated at constructor time

Reply:
removed pointer, but don't want to change the `report` return value to non-ptr, because for other resolvers the report is still optional. that can change once nursery is deleted completely